shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -257,7 +257,10 @@ - (BOOL)accessibilityPerformMagicTap {
   } else {
     view_.accessibilityElements = nil;
   }
-  UIAccessibilityPostNotification(UIAccessibilityLayoutChangedNotification, nil);
+  // TODO(goderbauer): If the previously focused element is still on screen we should fire a
+  //     UIAccessibilityLayoutChangedNotification instead. If the element is no longer on screen
+  //     we should use the semantics tree to figure out which element to focus instead.
+  UIAccessibilityPostNotification(UIAccessibilityScreenChangedNotification, nil);
```

Comment:
We can tell if the focused element is on the screen (i.e. has non-zero dimensions and is still in the semantics tree). Would it be worth checking for this in this patch?